src/TextBoxGroup/text-box-group-style.scss

```diff
@@ -16,37 +18,44 @@ $gray-500: #adadad;
 $transition-time: 200ms;
 .form-group.input-box-group {
     position: relative;
-    >label.custom-label:not(.form-check-label):not(.custom-checkbox):not(.custom-radio):not(.custom-control-label) {
-        display: block;
-        margin-bottom: 4px;
-        line-height: 1;
-        font-size: 12px;
-        font-weight: normal;
-    }
-    >.input-group {
+    > .input-group {
         position: relative;
-        >.input-box-group-wrapper {
+        > .input-box-group-wrapper {
+            color: $text;
+            will-change: border;
+            @include transition(border $transition-time / 2);
             position: relative;
             width: 100%;
             height: 42px;
-            border: 1px solid $gray-400;
+            border: 1px solid $grey6;
             border-radius: 4px;
             overflow: hidden;
             display: flex;
             align-items: center;
-            &:hover:not(.disabled) {
+            &:hover {
                 border-color: $blue;
+                background-color: white;
+            }
+            &:focus-within {
```

Comment:
`focus-within` is not widely supported, do you have a fallback method to cater for IE or old versions of browsers?

Reply:
There doesn't seem to be any fallback which can be done with css alone so I will leave this in for now. Thanks for pointing it out.